docs/docs/guides/check-permission.md

```diff
@@ -1,22 +1,27 @@
-# Check Permission
-
-There are two ways to check permission in the shield,
-
-1. REST/gRPC API
-2. Proxy Middleware
-
-## REST/gRPC API
-
-```sh
-POST v1beta1/check/{resourceId} HTTP/1.1
-Host: localhost:8000
-Content-Type: application/json
-
-{
-  "actionId": "read",
-  "namespaceId": "test-namespace"
-}
-```
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
+
+# Checking Pemrissions
+
+There are two ways to check a user permission on a resource in shield,
+## API Interface
+
+<Tabs groupId="api">
```

Comment:
Need to look into the rendering properly.

Reply:
This also has one `HTTP` tab only. Should we remove the tabs, where there is only one tab?